### PR TITLE
Remove SectionOnPage macro from es

### PR DESCRIPTION
**Pull request by Graywolf9**
### Description

Remove SectionOnPage macro from es

### Motivation

the chore of macro removal

### Related issues and pull requests

Relates to #11131

### CODE REVIEW

File: files/es/mdn/writing_guidelines/page_structures/macros/other/index.md [hunk 1]
```diff
@@ -37,5 +37,4 @@ Estas macros han sido reemplazadas por otras formas de hacer lo mismo y ya no de
 
 ### Enlaces
 
-- La macro [`SectionOnPage`](https://github.com/mdn/yari/blob/main/kumascript/macros/SectionOnPage.ejs) crea una frase que enlaza con el nombre de una sección y el artículo que contiene esa sección. Por ejemplo, `\{{SectionOnPage("/es/docs/Mozilla/Firefox/Releases/21", "Cambios para desarrolladores web")}}` genera lo siguiente: _{{SectionOnPage("/es/docs/ Mozilla/Firefox/Releases/21", "Cambios para desarrolladores web")}}_.
 - La macro [`Link`](https://github.com/mdn/yari/blob/main/kumascript/macros/Link.ejs) inserta un enlace a la página especificada en MDN, utilizando el título de la página como la cadena visible para hacer clic y la información sobre herramientas extraída del resumen de SEO de la página.
```
Comment (yin1999) [inline files/es/mdn/writing_guidelines/page_structures/macros/other/index.md:40]: Hi @Graywolf9, we can also remove this macro mention. As we have removed this macro from [mdn/yari](https://github.com/mdn/yari/blob/main/kumascript/macros). See also: english document

image

File: files/es/mozilla/firefox/releases/57/index.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ _No hay ningún cambio._
 
 #### Multimedia y WebRTC
 
-- Ahora se admiten mensajes de cualquier tamaño (hasta 1 GiB, aunque 256 kiB es más interoperativo) en {{domxref("RTCDataChannel")}} por medio del uso de la opción «end-of-record» (EOR) en los mensajes de SCTP. Consúltese {{SectionOnPage("/en-US/docs/Web/API/WebRTC_API/Using_data_channels", "Understanding message size limits")}} para obtener más información ({{bug(979417)}}).
+- Ahora se admiten mensajes de cualquier tamaño (hasta 1 GiB, aunque 256 kiB es más interoperativo) en {{domxref("RTCDataChannel")}} por medio del uso de la opción «end-of-record» (EOR) en los mensajes de SCTP. Consúltese [Understanding message size limits](/es/docs/Web/API/WebRTC_API/Using_data_channels#Understanding message size limits) para obtener más información ({{bug(979417)}}).
```
Comment (yin1999) [inline files/es/mozilla/firefox/releases/57/index.md:84]: The hash should be lowercase and concated with `_`

```suggestion
- Ahora se admiten mensajes de cualquier tamaño (hasta 1 GiB, aunque 256 kiB es más interoperativo) en {{domxref("RTCDataChannel")}} por medio del uso de la opción «end-of-record» (EOR) en los mensajes de SCTP. Consúltese [Understanding message size limits](/es/docs/Web/API/WebRTC_API/Using_data_channels#understanding_message_size_limits) para obtener más información ({{bug(979417)}}).
```

File: files/es/mdn/writing_guidelines/page_structures/macros/other/index.md (hunk 1 above)
Comment (Graywolf9) [reply files/es/mdn/writing_guidelines/page_structures/macros/other/index.md:40]: Thank you so much! I didn't notice that

File: files/es/mozilla/firefox/releases/66/index.md [hunk 3]
```diff
@@ -89,7 +89,7 @@ _Sin cambios._
...
-- En un paso en el camino hacia la implementación del soporte para la Política de funciones en una futura actualización de Firefox, {{domxref("MediaDevices.getUserMedia", "getUserMedia()")}} a no se puede usar en situaciones en las que no hay un origen adecuado para el contenido, como cuando se llama desde un espacio aislado {{HTMLElement("iframe")}} o desde una `data` URL ingresada en el Barra de direcciones por el usuario. Para obtener más detalles específicos, consulte {{SectionOnPage("/en-US/docs/Web/API/MediaDevices/getUserMedia", "Security")}} ({{bug(1371741)}}).
+- En un paso en el camino hacia la implementación del soporte para la Política de funciones en una futura actualización de Firefox, {{domxref("MediaDevices.getUserMedia", "getUserMedia()")}} a no se puede usar en situaciones en las que no hay un origen adecuado para el contenido, como cuando se llama desde un espacio aislado {{HTMLElement("iframe")}} o desde una `data` URL ingresada en el Barra de direcciones por el usuario. Para obtener más detalles específicos, consulte [Security](/es/docs/Web/API/MediaDevices/getUserMedia#Security) ({{bug(1371741)}}).
```
Comment (Graywolf9) [inline files/es/mozilla/firefox/releases/66/index.md:92]:
```suggestion
- En un paso en el camino hacia la implementación del soporte para la Política de funciones en una futura actualización de Firefox, {{domxref("MediaDevices.getUserMedia", "getUserMedia()")}} a no se puede usar en situaciones en las que no hay un origen adecuado para el contenido, como cuando se llama desde un espacio aislado {{HTMLElement("iframe")}} o desde una `data` URL ingresada en el Barra de direcciones por el usuario. Para obtener más detalles específicos, consulte [Security](/es/docs/Web/API/MediaDevices/getUserMedia#security) ({{bug(1371741)}}).
```

File: files/es/web/css/max-block-size/index.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ El tamaño máximo de la otra dimensión se especifica usando la propiedad {{css
 
 Esto es útil porque `max-width` se usa siempre para tamaños horizontales y `max-height` para tamaños verticales, y si necesita establecer longitudes basadas en el tamaño de su contenido de texto, debe ser capaz de hacerlo con la dirección de escritura en cuenta.
 
-En los momentos que normalmente utilizaría `max-height` o `max-width`, debería usar `max-block-size` para establecer el "alto" máximo del contenido (incluso si esto no es un valor vertical) y `max-inline-size` para establecer el "ancho" máximo del contenido (aunque esto puede ser vertical en lugar de horizontal). Vea el {{SectionOnPage("/es/docs/Web/CSS/writing-mode", "Ejemplo")}}, que muestra los diferentes modos de escritura en acción.
+En los momentos que normalmente utilizaría `max-height` o `max-width`, debería usar `max-block-size` para establecer el "alto" máximo del contenido (incluso si esto no es un valor vertical) y `max-inline-size` para establecer el "ancho" máximo del contenido (aunque esto puede ser vertical en lugar de horizontal). Vea el [Ejemplo](/es/docs/Web/CSS/writing-mode#Ejemplo), que muestra los diferentes modos de escritura en acción.
```
Comment (Graywolf9) [inline files/es/web/css/max-block-size/index.md:14]:
```suggestion
En los momentos que normalmente utilizaría `max-height` o `max-width`, debería usar `max-block-size` para establecer el "alto" máximo del contenido (incluso si esto no es un valor vertical) y `max-inline-size` para establecer el "ancho" máximo del contenido (aunque esto puede ser vertical en lugar de horizontal). Vea el [Ejemplo](/es/docs/Web/CSS/writing-mode#ejemplo), que muestra los diferentes modos de escritura en acción.
```

File: files/es/web/html/element/input/date/index.md [hunk 5]
```diff
@@ -58,7 +58,7 @@ La IU de la entrada generalmente varía entre navegadores; véase [Compatibiidad
 
 ## Valor
 
-Una {{domxref("DOMString")}} representando la fecha introducida en la entrada. La fecha es formateada siguiendo el ISO8601, descrito en {{SectionOnPage("/es/docs/Web/HTML/Date_and_time_formats", "Formato válido de una fecha en formato de texto")}}.
+Una {{domxref("DOMString")}} representando la fecha introducida en la entrada. La fecha es formateada siguiendo el ISO8601, descrito en [Formato válido de una fecha en formato de texto](/es/docs/Web/HTML/Date_and_time_formats#Formato válido de una fecha en formato de texto).
```
Comment (Graywolf9) [inline files/es/web/html/element/input/date/index.md:61]:
```suggestion
Una {{domxref("DOMString")}} representando la fecha introducida en la entrada. La fecha es formateada siguiendo el ISO8601, descrito en [Formato válido de una fecha en formato de texto](/es/docs/Web/HTML/Date_and_time_formats#formato_válido_de_una_fecha_en_formato_de_texto).
```

File: files/es/web/html/element/input/number/index.md [hunk 6]
```diff
@@ -82,7 +82,7 @@ El atributo `placeholder` es una cadena de texto que proporciona una pista corta
 
 Si el controlador de contenido tiene una direccionalidad ({{Glossary("LTR")}} o {{Glossary("RTL")}}), pero necesitas presentar el marcador de posición en la direccionalidad opuesta, puedes usar el algoritmo bidireccional para formatear caracteres Unicode para sobreescribir la direccionalidad del marcador de posición; véase [Cómo usar los controles Unicode para texto bidireccional](https://www.w3.org/International/questions/qa-bidi-unicode-controls) para más información.
 
-> **Nota:** Evita usar el atributo `placeholder` si puedes. No es semánticamente útil como otras formas de explicar tu formulario, y puede causar problemas técnicos imprevisto con tu contenido. Véase {{SectionOnPage("/es/docs/Web/HTML/Element/input", "Marcadores y parámetros de ejemplo")}} para más información.
+> **Nota:** Evita usar el atributo `placeholder` si puedes. No es semánticamente útil como otras formas de explicar tu formulario, y puede causar problemas técnicos imprevisto con tu contenido. Véase [Marcadores y parámetros de ejemplo](/es/docs/Web/HTML/Element/input#Marcadores y parámetros de ejemplo) para más información.
```
Comment (Graywolf9) [inline files/es/web/html/element/input/number/index.md:85]:
```suggestion
> **Nota:** Evita usar el atributo `placeholder` si puedes. No es semánticamente útil como otras formas de explicar tu formulario, y puede causar problemas técnicos imprevisto con tu contenido. Véase [Marcadores y parámetros de ejemplo](/es/docs/Web/HTML/Element/input#marcadores_y_parámetros_de_ejemplo) para más información.
```

File: files/es/web/html/element/input/text/index.md [hunk 7]
```diff
@@ -115,9 +115,9 @@ Consulta [Especificación de un patrón](#especificación_de_un_patrón) para ob
...
 
-Si el contenido del control tiene una direccionalidad ({{Glossary("LTR")}} o {{Glossary("RTL")}}) pero necesitas presentar el marcador de posición en la direccionalidad opuesta, puedes usar caracteres de formato de algoritmo bidireccional Unicode para anular la direccionalidad dentro del marcador de posición; consulta {{SectionOnPage("/es/docs/Web/Localization/Unicode_Bidirectional_Text_Algorithm", "Anulación de BiDi mediante caracteres de control Unicode")}} para esos caracteres.
+Si el contenido del control tiene una direccionalidad ({{Glossary("LTR")}} o {{Glossary("RTL")}}) pero necesitas presentar el marcador de posición en la direccionalidad opuesta, puedes usar caracteres de formato de algoritmo bidireccional Unicode para anular la direccionalidad dentro del marcador de posición; consulta [Anulación de BiDi mediante caracteres de control Unicode](/es/docs/Web/Localization/Unicode_Bidirectional_Text_Algorithm#Anulación de BiDi mediante caracteres de control Unicode) para esos caracteres.
```
Comment (Graywolf9) [inline files/es/web/html/element/input/text/index.md:118]:
```suggestion
Si el contenido del control tiene una direccionalidad ({{Glossary("LTR")}} o {{Glossary("RTL")}}) pero necesitas presentar el marcador de posición en la direccionalidad opuesta, puedes usar caracteres de formato de algoritmo bidireccional Unicode para anular la direccionalidad dentro del marcador de posición; consulta [Anulación de BiDi mediante caracteres de control Unicode](/es/docs/Web/Localization/Unicode_Bidirectional_Text_Algorithm#anulación_de_bidi_mediante_caracteres_de_control_unicode) para esos caracteres.
```

File: files/es/web/html/element/input/text/index.md [hunk 8]
```diff
@@ -115,9 +115,9 @@ Consulta [Especificación de un patrón](#especificación_de_un_patrón) para ob
...
 
-> **Nota:** Evita utilizar el atributo {{HTMLAttrDef("placeholder")}} si puedes. No es tan útil semánticamente como otras formas de explicar tu formulario y puede causar problemas técnicos inesperados con tu contenido. Consulta {{SectionOnPage("/es/docs/Web/HTML/Element/input", "Etiquetas y marcadores de posición")}} para obtener más información.
+> **Nota:** Evita utilizar el atributo {{HTMLAttrDef("placeholder")}} si puedes. No es tan útil semánticamente como otras formas de explicar tu formulario y puede causar problemas técnicos inesperados con tu contenido. Consulta [Etiquetas y marcadores de posición](/es/docs/Web/HTML/Element/input#Etiquetas y marcadores de posición) para obtener más información.
```
Comment (Graywolf9) [inline files/es/web/html/element/input/text/index.md:120]:
```suggestion
> **Nota:** Evita utilizar el atributo {{HTMLAttrDef("placeholder")}} si puedes. No es tan útil semánticamente como otras formas de explicar tu formulario y puede causar problemas técnicos inesperados con tu contenido. Consulta [Etiquetas y marcadores de posición](/es/docs/Web/HTML/Element/input#etiquetas_y_marcadores_de_posición) para obtener más información.
```

**Review by JuanVqz — APPROVED** [review]
Gracias @Graywolf9 :rocket:

**Pull request merged**